googleapiclient/discovery.py

```diff
@@ -950,6 +951,17 @@ def _set_service_methods(self):
     self._add_next_methods(self._resourceDesc, self._schema)
 
   def _add_basic_methods(self, resourceDesc, rootDesc, schema):
+    # If this is the root Resource, add a new_batch_http_request() method.
+    if resourceDesc == rootDesc:
+      def new_batch_http_request(callback=None):
+        batchPath = 'batch'
+        if 'batchPath' in rootDesc:
+          batchPath = rootDesc['batchPath']
+        return BatchHttpRequest(
+          callback = callback,
```

Comment:
No space on either side of an equals sign used to specify a keyword argument. (On this line and the next line.)